lib/compiler/doc/src/compile.xml

```diff
@@ -620,7 +620,21 @@ module.beam: module.erl \
             option is internal to the compiler and can be changed or
             removed at any time without prior warning.</p>
           </item>
-	</taglist>
+
+          <tag><c>line_coverage</c></tag>
+          <item>
+            <p><marker id="line_coverage"/>
+            Instrument the compiled code for line coverage.</p>
```

Comment:
Feels like there needs to be some links to docs explaing when/how to use this option?

Reply:
Yes, done.